File: C++/1157.cpp
#include <iostream>
using namespace std;

int main(){
    int i = 1, r = 0, num = 0;
    cin >> num;
    for(i = 1; i <= num; i++){
        r = num % i;
        if(r == 0){
            cout << i << endl;
        }
    }
    return 0;
}

File: java/1157.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int i = 1, r = 0, num = 0;
        num = in.nextInt();
        for(i = 1, r = 0; i <= num; i++){
            r = num % i;
            if(r == 0){
                System.out.println(i);
            }
        }
    }
}